### PR TITLE
OSSMDOC-581 Update load test results topic

### PR DESCRIPTION
**Pull request by rh-tokeefe**
Version(s): 4.6-11

Issue: https://issues.redhat.com/browse/OSSMDOC-581

Link to docs preview: http://file.bos.redhat.com/tokeefe/OSSMDOC-581/service_mesh/v2x/ossm-performance-scalability.html#ossm-load-test-results_performance-scalability

Eng review: jwendell
QE review: FillipB, pbajjuri20
Peer review: jstickler

### CODE REVIEW

**Review by jwendell — APPROVED** [review]
LGTM. Perhaps we should somehow *__emphasize__* that this is Istio upstream and not OSSM? I know it's not the scope of this PR, just a thought for maybe a future change.

**Comment by FilipB** [comment]
LGTM

File: modules/ossm-load-test-results.adoc [hunk 1]
```diff
@@ -52,7 +52,7 @@ The latency, throughput, and the proxies' CPU and memory consumption are measure
 
 === CPU and memory consumption
 
-Since the sidecar proxy performs additional work on the data path, it consumes CPU and memory. As of Istio 1.1, a proxy consumes about 0.6 vCPU per 1000 requests per second.
+Since the sidecar proxy performs additional work on the data path, it consumes CPU and memory. As of Istio 1.12.3, a proxy consumes about 0.5 vCPU per 1000 requests per second.
```
Comment (pbajjuri20) [inline modules/ossm-load-test-results.adoc:55]: https://istio.io/latest/docs/ops/deployment/performance-and-scalability/#cpu-and-memory

Here, the Istio version is 1.14.1

**Review by pbajjuri20 — APPROVED** [review]

File: modules/ossm-load-test-results.adoc [hunk 2]
```diff
@@ -7,11 +7,11 @@ This module included in the following assemblies:
 = Load test results
 
 The upstream Istio community load tests mesh consists of *1000* services and *2000* sidecars with 70,000 mesh-wide requests per second.
-Running the tests using Istio 1.6.8, generated the following results:
+Running the tests using Istio 1.12.3, generated the following results:
```
Comment (pbajjuri20) [inline modules/ossm-load-test-results.adoc:10]: Istio version 1.14.1?

File: modules/ossm-load-test-results.adoc (hunk 1 above)
Comment (jwendell) [reply modules/ossm-load-test-results.adoc:55]: @pbajjuri20 Make sure to look at the upstream 1.12 doc page: https://istio.io/v1.12/docs/ops/deployment/performance-and-scalability/

File: modules/ossm-load-test-results.adoc (hunk 2 above)
Comment (jwendell) [reply modules/ossm-load-test-results.adoc:10]: @pbajjuri20 Make sure to look at the upstream 1.12 doc page: https://istio.io/v1.12/docs/ops/deployment/performance-and-scalability/

**Review by pbajjuri20 — CHANGES_REQUESTED** [review]
It looks good to me.

File: modules/ossm-load-test-results.adoc (hunk 2 above)
Comment (pbajjuri20) [reply modules/ossm-load-test-results.adoc:10]: @jwendell : thanks for the clarification

**Review by pbajjuri20 — APPROVED** [review]

**Comment by rh-tokeefe** [comment]
@JStickler this PR ready for peer review.

**Comment by rh-tokeefe** [comment]
@JStickler this is ready for peer review.

**Review by JStickler — APPROVED** [review]
LGTM

**Pull request merged**